packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -1545,6 +1546,99 @@ curl -v -X POST \
 }
 ```
 
+#### Enroll WebAuthn Factor
+
+Enrolls a user with a WebAuthn factor. The enrollment process starts with getting the WebAuthn credential creation options, which are used to help select an appropriate authenticator using the WebAuthn API.
```

Comment:
that rather than which here:

Enrolls a user with a WebAuthn factor. The enrollment process starts with getting the WebAuthn credential creation options that are used to help select an appropriate authenticator using the WebAuthn API.